### ISSUE TITLE
Usecase - adding a new compartment (Lysosome) to each organ

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1319]
The new concept makes it even more cumbersome than the old one. We should brainstorm a bit if we can find a better workflow.

**Comment by Yuri05** [comment]
😲

**Comment by PavelBal** [comment]
Property "Do not overwrite container"?

**Comment by Yuri05** [comment]
I see so many potential problems... needs to be discussed.

**Comment by msevestre** [comment]
I am working on this issue here
https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/2240

Can you attach the file @PavelBal so that I can test your behavior and describe what you would want to have happen?

**Comment by msevestre** [comment]
>I see so many potential problems... needs to be discussed.

I see so many opportunities :)

**Comment by PavelBal** [comment]
[Lysosome.zip](https://github.com/Open-Systems-Pharmacology/MoBi/files/15292107/Lysosome.zip)
In the attached project, there are three modules:

- **iv 0.005 mg/kg (2 min)**: A PK-Sim module with the full body PBPK
- **Lysosome_Bone**: A module with a "Lysosome"-compartment that will be added to the "Bone" organ (parent path: `Organism|Bone`). This is how it works now.

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/e1ceceaf-3b0f-4455-aafb-0d01596bfd23)

It also contains a (new) neighborhood between bone cells and bone lysosome:

![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/913783bb-3ea0-49f8-a31e-49e06accf5df)


With this concept, I can only add the new compartment to one organ (here - bone): Now what I *want* to do is to add a lysosome compartment to ALL organs. This could work with the concept of merging (instead of fully overwriting) the modules. THe **Lysosome_all** module is an example how it could work.

- It contains the organ structure of the whole organism (I manually created all the relevant organs, including the mucosa... Probably would even work if I just save the whole organism as a module, but I wanted it to be clean). One created, this module can be stored as a backbone for other modules.
- I loaded the Lysosome compartment in each organ.
  - I had to manually rename the neighborhood every time because the default name "Bone_int_Bone_lys" is stored in the pkml... But that's okay, done only once.
  - I had to manually correct the neighbors for the neighborhoods. This was annoying, but I don't know what a good solution would be.
  
  
![image](https://github.com/Open-Systems-Pharmacology/MoBi/assets/25903040/11272a0a-fb50-4948-8862-92d6826d9536)
